utils.py

```diff
@@ -420,6 +437,14 @@ def get_human_readable_time_string(time_msec):
 def are_datetimes_close(later_datetime, earlier_datetime):
     """Given two datetimes, determines whether they are separated by less than
     feconf.PROXIMAL_TIMEDELTA_SECS seconds.
+
+    Args:
+        later_datetime: datetime. Later date and time.
+        earlier_datetime: datetime. Earlier date and time.
+
+    Returns:
+        datetime. Whether they are seprated by less than
```

Comment:
Incorrect type?

Reply:
@DubeySandeep I'm not sure actually like if we are comparing two DateTime format than result should be in DateTime too, no??

Reply:
I think line 450 will help you understand the return type OR you can use the print statement to log the type of return value.

Reply:
done